notifier/notifier.go

```diff
@@ -93,7 +93,7 @@ func (a *Alert) Resolved() bool {
 	return a.ResolvedAt(time.Now())
 }
 
-// ResolvedAt returns true off the activity interval ended before
+// ResolvedAt returns true iff the activity interval ended before
```

Comment:
```suggestion
// ResolvedAt returns true if the activity interval ended before
```

Reply:
iff = if and only if so the change is correct.